internal/shim-sgx/src/handler/enarx.rs

```diff
@@ -1,8 +1,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
+use sallyport::request;
+use sallyport::syscall::SGX_DUMMY_QUOTE;
+use sallyport::syscall::SGX_DUMMY_TI;
+use sallyport::syscall::SYS_ENARX_GETATT;
 use sallyport::syscall::{BaseSyscallHandler, EnarxSyscallHandler, SGX_QUOTE_SIZE, SGX_TECH};
 use sallyport::untrusted::{UntrustedRef, UntrustedRefMut, ValidateSlice};
-use sallyport::Reply;
+
+use crate::uarch::REPORT;
+use crate::uarch::TARGETINFO;
```

Comment:
Is there a specific reason these are capitalized? If that was something specific to that particular submodule, how about we just rename these to "normal" Rust struct names (i.e. `Report` and `TargetInfo`) to avoid confusion?